proposals/lip-0047.md

```diff
@@ -498,11 +498,147 @@ This module does not define any specific logic for other modules.
 
 This module does not define any endpoint for off-chain services.
 
-### Protocol Logic During Block Lifecycle
+### Genesis Block Processing
 
-#### After Genesis Block Execution
+The following two steps are executed as part of the genesis block processing, see the “Update genesis block schema and processing” LIP for details. 
 
-After the genesis block `g` is executed, the following logic is executed:
+#### Genesis State Initialization
+
+To initialize the state of a genesis block `g`, the following logic is executed. If any step fails, the block is discarded and has no further effect.
+
+Let `genesisBlockAssetBytes` be the bytes included in the block assets for the PoA module and let `genesisBlockAssetObject` be the deserialization of `genesisBlockAssetBytes` according to the `genesisPoAStoreSchema` schema, given below, then: 
```

Comment:
Consider using a short variable name such as `genesisAsset` or `assetPoA` instead of `genesisBlockAssetObject` for better readability and compactness below.

Reply:
Added a correct formulation for the deserialization of the assets as Andreas suggested
Also, shortened `genesisBlockAssetObject` to `assetPoA` as Jan suggested

Reply:
can be resolved

Reply:
Can be resolved.